### PR TITLE
DI-198 Performance Tests ADR & Prototype

### PR DESCRIPTION
**Pull request by JackPlowman**
## Link to JIRA Ticket

- https://nhsd-jira.digital.nhs.uk/browse/DI-198

## Description

This PR has the Aritcheture Decision Record and prototype for the performance tests

## Type of change

- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

## Developer Checklist

- [x] I have performed a self-review of my own code
- [x] I have run the [code formatting checks](../README.md#code-quality)
- [x] I have run the [code quality checks](../README.md#code-quality)
- [x] New code meets [standards](https://nhsd-confluence.digital.nhs.uk/display/DI/DI+Ways+of+Working) agreed by the team
- [x] I have made corresponding changes to the documentation

## Code Reviewer Checklist

- [ ] I can confirm the changes have been tested or approved by a tester

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -209,6 +210,20 @@ This testing includes:
...
+This testing includes:
+
+- Checking non functional requirements (performance/reliability)
+
+This testing is generally done by a tester
+
+To run the performance tests run the following commands
+
+    make tester-build
+    make performance-test
```
Comment (mabe13) [inline README.md:226]: Just need some text to explain this target need the developer to be mfa'ed and why.

File: README.md [hunk 2]
```diff
@@ -194,12 +195,12 @@ This testing is generally done by a developer
 
 To run components tests run the following commands
 
-    make tester-build
+    make tester-build mock-dos-db-setup
```
Comment (mabe13) [inline README.md:198]: Just need some text to explain the `mock-dos-db-setup` target need the developer to be mfa'ed and why.

**Review by mabe13 — CHANGES_REQUESTED** [review]
Other than the small README changes all good 👍

File: README.md (hunk 2 above)
Comment (JackPlowman) [reply README.md:198]: Updated with comment to sign into non-prod

File: README.md (hunk 1 above)
Comment (JackPlowman) [reply README.md:226]: Updated with comment to sign into non-prod

**Review by mabe13 — APPROVED** [review]

**Review by mik3ola — APPROVED** [review]

**Pull request merged**